presto-iceberg/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.iceberg.version>0.14.0</dep.iceberg.version>
+        <dep.iceberg.version>0.14.1</dep.iceberg.version>
```

Comment:
Is this needed?

Reply:
Apparently it's not needed. I've rebuilt and tested the project without it.